commands/deploy.go

```diff
@@ -31,7 +31,7 @@ type DeployCmd struct {
 	AppID          int           `required short:"i" help:"AppID to deploy application to."`
 	Environment    string        `short:"e" default:"production" help:"Environment to deploy application to."`
 	Debug          bool          `help:"Display extra debugging information about what is happening inside sectionctl."`
-	Directory      string        `short:"C" default:"." help:"Directory which contains the application to deploy."`
```

Comment:
why?

Reply:
I think this was copy-pasted from your PR. Seems like we could leave the shorthand flag in place, I'll add it back

Reply:
ah. yeah it probably was giving me some sort of error. not sure if it was at a local dev, compilation, or runtime level issue though.